at-hoangtran/Week3/Day3/Exercise4-Day2.rb

```diff
@@ -0,0 +1,5 @@
+arr = [2, 2, 1, 1, 5, 3, 5]
+if arr.uniq!
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/a5ed3ba5b6a204f5dbf8b2b54972d06d02f2dcdb/at-hoangtran/Week3/Day3/Exercise4-Day2.rb#L2"></span>Style/IfUnlessModifier: Favor modifier <code>if</code> usage when having a single-line body. Another good alternative is the usage of control flow <code>&amp;&amp;</code>/<code>||</code>. (https://github.com/rubocop-hq/ruby-style-guide#if-as-a-modifier)</td>
    </tr>
  </tbody>
</table>